tensorflow/contrib/lite/kernels/gather_test.cc

```diff
@@ -48,8 +48,8 @@ class GatherOpModel : public SingleOpModel {
     PopulateStringTensor(input_, data);
   }
 
-  void SetPositions(std::initializer_list<int32> data) {
```

Comment:
Is this change relevant? If it's not, it's preferred to revert this and only put relevant changes in a PR.

Reply:
I cannot compile this file if I neither `using ::tensorflow::int32;` nor change `int32` to `int32_t`.

My environment is OSX 10.12.6 with clang++ version "900.0.39.2". The test is executed by:

```shell
bazel test -s -c dbg //tensorflow/contrib/lite/kernels:gather_test
```

Can you help verify this? Thanks.

Reply:
I see. Indeed the test is broken on OSX. 
Let change this to `int` instead of `int32_t` to align with other files in the same directory.